benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1169,17 +1170,22 @@ void GraphicsBenchmarkApp::DrawExtraInfo()
 
     uint64_t frequency = 0;
     PPX_CHECKED_CALL(GetGraphicsQueue()->GetTimestampFrequency(&frequency));
-    const float gpuWorkDurationInSec = static_cast<float>(mGpuWorkDuration / static_cast<double>(frequency));
-    const float gpuWorkDurationInMs  = gpuWorkDurationInSec * 1000.0f;
+    const float gpuWorkDurationInSec      = static_cast<float>(mGpuWorkDuration / static_cast<double>(frequency));
+    const float gpuWorkDurationInMs       = gpuWorkDurationInSec * 1000.0f;
+    const float gpuWorkDurationMAInSec    = static_cast<float>(mWeightedFrameTimeTicks.Value() / static_cast<double>(frequency));
+    const float gpuWorkDurationMAInMs     = gpuWorkDurationMAInSec * 1000.0f;
+    const float gpuWorkDurationMAStdInSec = static_cast<float>(std::sqrt(mWeightedFrameTimeTicks.Variance()) / static_cast<double>(frequency));
```

Comment:
why do you need to cast frequency to double if all others are float?

Reply:
I'm confused as well, it's copy pasted from previous lines, and I've yet to figure out why it's written this way.

Currently this PR is a proposal, so I've not cleaned everything up.